src/ig/IGExporter.ts

```diff
@@ -552,18 +586,19 @@ export class IGExporter {
     const pageData = pages.map(page => {
       const nameParts = page.match(/^(\d+)_(.*)/);
       let prefix: number = null;
-      let name: string;
+      let title: string;
```

Comment:
I kind of find `title` confusing, as I expect it is the human-readable title.  But it's not.  It's just the name without the numeric prefix.  How about calling what you call `nameWithPrefix` just `name` (since that's the one that will survive past deprecation) and call what you currently call `title` `nameWithoutPrefix`.  What do you think?